README.md

```diff
@@ -51,3 +51,32 @@ groups to Dependency Track permissions.
 ## Maintenance instructions
 
 [Can be found on Confluence.](https://companieshouse.atlassian.net/wiki/spaces/DEV/pages/4601348098/Maintaining+Dependency+Track)
+
+## Populating Dependency Track
+
+### Using script: [populate-dependency-track](./bin/populate-dependency-track)
+
+The script iterates over a list of repositories provided in a CSV file defining
+the repository name and language. It will then clone the repository locally
+before running the Concourse task to generate the SBOM and send to Dependency
+Track.
+
+**Example usage:**
+
+```sh
+$ cat ./repos
+overseas-entities-api,java
```

Comment:
Are there repos in more than one language? Assuming that this is so, for the sake of the design of this PR, could we find a way of coping?

Reply:
Maybe, but I'll implement if there are any repos where this is actually the case.